mikado-app/src/pages/Graph/DisplayLayer.js

```diff
@@ -45,15 +47,30 @@ function DisplayLayerInternal({ uid, notifySuccessElseError, fabNotifySuccessEls
   // Workaround to run fitView on the next render after the store is updated
   useEffect(() => {
     // Yield the event loop so that React Flow can receive the nodes before telling it to fit them.
-    const id = setTimeout(() => {
-      fitView();
+    setDisplayLayerHandle(new DisplayLayerHandle(operations, nodes.length !== 1 ? void 0 : nodes[0].id));
+  }, [loadAutoincremented]);
 
-      // Set the display handle immediately after load
-      setDisplayLayerHandle(new DisplayLayerHandle(operations, nodes.length !== 1 ? void 0 : nodes[0].id));
-    }, 5); // Set to > 0 due to it not fitting sometimes, delay same as before
-    return () => clearTimeout(id);
-  }, [fitView, loadAutoincremented]);
+  // Zoom on transition call
+  useEffect(() => {
+    if (graphTransition.transitionIn === true && graphTransition.transitionOut === false) {
```

Comment:
TODO: disable the transitions. @Daanish-Khan mentioned in the chat that there are some bugs here.

Reply:
Yeah it's kinda scuffed. Current bugs havent been found, but impl is prone to bugs if **any** lag occurs during transition. I want the anims to be not dependant on async timeouts and instead on (placeholder names) TransitionStart, TransitionLoadGraph, TransitionEnd, where end of start anim loads the graph, then when the graph is loaded to trigger the end transition